local-cli/generator-android/templates/src/app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,7 @@
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
+        android:windowSoftInputMode="adjustResize"
```

Comment:
I've found that using `adjustPan` instead of `adjustResize` works better for scrolling the input area to the top of the keyboard.